cassandra/conf.yaml.example

```diff
@@ -2,6 +2,9 @@ instances:
   - host: localhost
     port: 7199
     cassandra_aliasing: true
+    tags:
+      environment: default_environment
+      datacenter: default_datacenter
```

Comment:
I am currently writing an article for Datadog blog, this will be in there most definitely. Adding a short comment though, this indeed matters a lot, good catch.